linked_list/LinkedList.c

```diff
@@ -0,0 +1,233 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+struct Node {
+	int d;			//integer type data
+	struct Node* next;	// pointer to same structure
+};
+
+//  Creatig node
+struct Node* getNewNode(int data) {
+	struct Node *nw_node = NULL;
+	while (nw_node == NULL) {
+		nw_node = (struct Node*) malloc(sizeof(struct Node));	//  nw_node a Node to be added
+	}
```

Comment:
Please just return `NULL` if malloc fails. This can be harmful if such situation actually occurs.